pkg/reconciler/vertex/scaling/scaling.go

```diff
@@ -309,27 +312,35 @@ func (s *Scaler) scaleOneVertex(ctx context.Context, key string, worker int) err
 		if diff > maxAllowed {
 			diff = maxAllowed
 		}
+		if secondsSinceLastScale < scaleDownCooldown {
```

Comment:
move it after 176?

Reply:
why?

Reply:
aren't we doing the check there, and this equality for change?

Reply:
This check happens after we know it needs to scale down.